edu.wpi.first.wpilib.plugins.java/src/main/resources/java-zip/ant/build.properties

```diff
@@ -1,7 +1,10 @@
 # Deployment information
 username=lvuser
 password=
+adminUsername=admin
+adminPassword=
 deploy.dir=/home/lvuser
+libDeploy.dir=/usr/local/frc/lib
```

Comment:
It is used for .so's which someone has chosen not to already wrap up in their JAR and load like WPILib loads the JNI one. 3rd party JARs are bundled with the user program JAR right now. I have not checked to verify that it is on java.library.path (where Java looks for libraries on a System.loadLibrary() call) but I believe it should be.